entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityAccessControlHandler.kt

```diff
@@ -121,52 +97,27 @@ class EntityAccessControlHandler(
         @RequestHeader httpHeaders: HttpHeaders,
         @RequestParam params: MultiValueMap<String, String>
     ): ResponseEntity<*> {
-        val count = params.getFirst(QUERY_PARAM_COUNT)?.toBoolean() ?: false
-        val (offset, limit) = extractAndValidatePaginationParameters(
-            params,
-            applicationProperties.pagination.limitDefault,
-            applicationProperties.pagination.limitMax,
-            count
-        )
```

Comment:
weird to have this since this is part of the query params refactoring feature ... missing a rebase maybe?

Reply:
when I do a rebose on the query params refactoring feature I probably forgot this endpoint